src/js/AccessibilityUtils.js

```diff
@@ -720,11 +719,13 @@ axs.utils.getAriaPropertyValue = function(propertyName, value, element) {
             result.value = validNumber.value;
         }
         return result;
+    case "decimal":
     case "number":
         var validNumber = axs.utils.isValidNumber(value);
-        if (validNumber.valid) {
-            result.valid = true;
-            result.value = validNumber.value;
+        if (!validNumber.valid) {
+            result.valid = false;
+            result.reason = validNumber.reason;
+            return result;
         }
```

Comment:
It also seems weird that we don't have a return for the valid case here (undoubtedly my fault) - mind adding one?